### ISSUE TITLE
Error on Ubuntu 18 : undefined symbol: g_log_structured_standard

### ISSUE THREAD

**Issue by AniruddhPurohit** [justadudewhohacks/opencv4nodejs#582]
I m getting error when starting application
----------------------------------------------------------------
**Ubuntu 18.04 LTS
node 8.11.3
npm 6.4.1**

```
internal/modules/cjs/loader.js:718
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: /usr/lib/x86_64-linux-gnu/libatk-1.0.so.0: undefined symbol: g_log_structured_standard
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:718:18)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/Ani/myExample/node_modules/opencv4nodejs/lib/cv.js:17:8)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

I don't understand the actual cause of this problem.

**Comment by steveunixson** [comment]
It basically says that you are missing libatk-1.0.so.0 on your system. 
So just do:
`sudo apt install libatk1.0-0
And then rebuild the library just by
`npm r opencv4nodejs
`npm i opencv4nodejs
inside of your project directory

**Comment by Simon1059770342** [comment]
> It basically says that you are missing libatk-1.0.so.0 on your system.
> So just do:
> `sudo apt install libatk1.0-0`
> And then rebuild the library just by
> `npm r opencv4nodejs`
> `npm i opencv4nodejs`
> inside of your project directory
> Also to check what's libs it's missing you can do:
> `ldd path/to/your/project/opencv4nodejs/build/Release/opencv4nodejs.node | grep "not found"`
> If the result is empty it's all good

npm WARN checkPermissions Missing write access to /www/wwwroot/http/capture_back/node_modules/isarray
npm WARN checkPermissions Missing write access to /www/wwwroot/http/capture_back/node_modules/util-deprecate
npm WARN checkPermissions Missing write access to /www/wwwroot/http/capture_back/node_modules/readable-stream
npm WARN cctv@1.0.0 No description
npm WARN cctv@1.0.0 No repository field.

npm ERR! path /www/wwwroot/http/capture_back/node_modules/isarray
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall access
npm ERR! enoent ENOENT: no such file or directory, access '/www/wwwroot/http/capture_back/node_modules/isarray'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-07-10T06_35_19_856Z-debug.log
Following your steps, I got the error above.

**Comment by steveunixson** [comment]
@Simon1059770342 
Your path is incorrect and it's very strange...
/www/wwwroot/http/capture_back/ obviously doesn't exists.
maybe you are trying to run npm under a root user?
Maybe you should consider reinstall node using nvm
https://github.com/nvm-sh/nvm

**Comment by nwheatle** [comment]
I had a similar error - and it all turned out to be due to the fact that I had changed node versions in the middle of developing my app (probably!). I had installed my npm packages using a different version than when I was running the app. 

I just  uninstalled all my dependencies

`npm uninstall express opencv4nodejs canvas body-parser ...
`
I switched to the node version I wanted to use (the latest version)
`nvm install --lts
`
then I re-installed them again

`npm install express opencv4nodejs canvas body-parser ...`

Then the errors stopped!
